ArduSub/joystick.cpp

```diff
@@ -660,8 +662,8 @@ void Sub::default_js_buttons()
         {JSButton::button_function_t::k_mount_center,           JSButton::button_function_t::k_none},
 
         {JSButton::button_function_t::k_input_hold_set,         JSButton::button_function_t::k_none},
-        {JSButton::button_function_t::k_mount_tilt_down,        JSButton::button_function_t::k_none},
-        {JSButton::button_function_t::k_mount_tilt_up,          JSButton::button_function_t::k_none},
+        {JSButton::button_function_t::k_mount_tilt_down,        JSButton::button_function_t::k_mount_pan_right},
```

Comment:
These are the wrong way around. left->right and right->left please.

Reply:
Done.